src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventSchema.scala

```diff
@@ -79,6 +79,37 @@ object CassandraPopularTopics {
   }
 }
 
+object CassandraConjunctiveTopics {
+  def apply(item: Event): Seq[ConjunctiveTopicAggregate] = {
+    val keywords = item.computedfeatures.keywords
+    val keywordPairs = keywords.combinations(2).flatMap(combination=>Seq(
```

Comment:
If there's a single keyword, the `combinations(2)` call returns an empty sequence. So, the combination in the `for` loop below short-circuits and this method emits an empty sequence.